FastpackDevserver/WebsocketHandler.re

```diff
@@ -0,0 +1,143 @@
+module WServer = Websocket_cohttp_lwt;
```

Comment:
The code in this file is mostly borrowed and inspired by https://github.com/actionshrimp/ocaml-livereload. I'm not sure about the license here so maybe it should be further from the original.